api-reference/v2/api-reference-guide.md

```diff
@@ -0,0 +1,213 @@
+# API reference guide
+
+This `api-reference-guide` document provides extra writing tips describing **HOW** to fill in each of the sections within the [`api-reference-template`](api-reference-template.md) file.
+
+- _Document version_: 0.0.1
+- _Last updated_: June 27, 2022
+
+## Overview
+
+API references are technical manuals that provide the specifications and integration instructions of APIs. An accurate, concise, well-structured API documentation facilitates the adoption of APIs and increases the overall user experience.
+
+The [`api-reference-template`](api-reference-template.md) is designed to help you build the API documentation efficiently and keep it consistent in both format and appearance.
+
+## Before writing the API references
+
+- Familiarize yourself with the API protocol. If possible, try making some API calls in a testing environment. This API reference template works best with the REST API protocol. If you are working with other protocols, you may need to adjust the organization of some sections.
```

Comment:
Hi. I am reading this information for the first time.
I think you could break up some of the sentences in the `Before you start` section and possibly create a separate section for testing the API?
Do you recommend that authors should read and understand the API specification (and components of the API specification)?

Reply:
Hi Karen, thanks for the comment! The idea sounds good but at this point, we are not focussing on providing links to learn API documentation. Maybe we will put some references in the deep dive document that we are working on

> Hi. I am reading this information for the first time. I think you could break up some of the sentences in the `Before you start` section and possibly create a separate section for testing the API? What tools or clients could you use to test the API and help you document the endpoints? Do you recommend that authors should read and understand the API specification (and components of the API specification)?